.github/workflows/notify-issue-comment.yml

```diff
@@ -0,0 +1,20 @@
+name: Issue Comment Slack Notifier
+
+on:
+  issue_comment:
```

Comment:
Based on the doc:
`The issue_comment event occurs for comments on both issues and pull requests.`
If we want that workflow to trigger only for the comment on issue, we need to add additional check:
`if: ${{ !github.event.issue.pull_request }}`

But I'm thinking that comments on PR could be beneficial too. We could have a separate statement for comments on PR with the comment itself.

Reply:
I think it would be good to have Slack messages on both. Do you think any changes are needed here?

Reply:
If we are going to use both, then small nitpick: let's change the name of workflow to reflect that.

Separately, I've tried to post link on the comments for an issue and for a PR. In case of PR's comment slack is not trying to build preview. Let's add separate statement for PR's comments to include the comment text.

Reply:
We should be notified for both pull request comments and issue comments. I think we are a bit confused as we haven't seen examples of how these notifications would look like. Is it possible to have a couple of test notifications on the team's channel?